definitions/marts/looker_studio_marts/ls_declarations_provider_names.sqlx

```diff
@@ -9,40 +9,39 @@ config {
     },
     bigquery: {

```

Comment:
Same approval question.

Reply:
It is the same as the V2 file you approved previously.